module3/src/config.ts

```diff
@@ -0,0 +1,4 @@
+export const Config = {
+    port: 3000,
+    databaseURL: 'postgres://tbrebkof:ybWL2zwMXJJLQqk8GaW-eWucW0EMa-nm@mel.db.elephantsql.com/tbrebkof'
```

Comment:
move it to env file. It is a private info

Reply:
Moved to .env